acquia/blt
# Allows Drush ANSI output to be disabled.

Changes proposed:
---------
- Allow Drush's ANSI output to be disabled.

On environments which don't support ANSI output (looking at you Acquia Cloud), the output generated by Drush calls dispatched through BLT can be pretty ugly. For example, 
```
[Acquia\Blt\Robo\Tasks\DrushTask] Running /var/www/html/<env>/vendor/bin/drush @self cache-rebuild --no-interaction -v --ansi in /var/www/html/<env>/docroot
 [37;46;1m[success][39;49;22m Cache rebuild complete.
```

When the logs are full of this it's pretty distracting. Currently there is no way to disable this, because the `--ansi` flag is passed during every Drush call. This PR allows that to be configured.

Steps to replicate the issue:
----------
1.  Deploy to an environment that doesn't support ANSI output and witness the results.

Steps to verify the solution:
-----------
1. Apply this patch.
2. In `blt.yml` set `drush.ansi` to `false`.
3. Deploy and confirm the output no longer contains ANSI sequences.
